src/gt4py/next/ffront/past_to_itir.py

```diff
@@ -382,12 +382,11 @@ def visit_BinOp(self, node: past.BinOp, **kwargs) -> itir.FunCall:
         )
 
     def visit_Call(self, node: past.Call, **kwargs) -> itir.FunCall:
-        if node.func.id in ["maximum", "minimum"] and len(node.args) == 2:
+        if node.func.id in ["maximum", "minimum"]:
+            assert len(node.args) == 2
             return itir.FunCall(
                 fun=itir.SymRef(id=node.func.id),
                 args=[self.visit(node.args[0]), self.visit(node.args[1])],
             )
         else:
-            raise AssertionError(
-                "Only 'minimum' and 'maximum' builtins supported supported currently."
-            )
+            raise AssertionError("Only 'minimum', and 'maximum' builtins supported currently.")
```

Comment:
Completely optional suggestion since this is unrelated to this PR but since the code has been touched anyway maybe a `NotImplementedError` would make more sense here. Feel free to ignore.
```suggestion
            raise NotImplementedError("Only 'minimum', and 'maximum' builtins supported currently.")
```